src/Sarif/Microsoft.Json.Schema.ToDotNet.targets

```diff
@@ -32,22 +32,6 @@
     <!-- Build the object model if necessary. -->
     <CallTarget Targets="BuildObjectModel" />
 
```

Comment:
we should consider leaving this hack if we add it again. I predict we will continue to find it useful to hand-tune some files while we wait on json fixes. it's easy enough to leave the targets file in and just delete all files from notyetautogen.